### ISSUE TITLE
Retired Documents shouldn't be automatically treated as "Do Not Read"

### ISSUE THREAD

**Issue by fantasai** [w3c/tr-design#180]
The styling of the warning on 
  https://www.w3.org/TR/css-print/
and
  https://www.w3.org/TR/css-speech/
is inappropriately obnoxious. Retiring a REC track document to NOTE status is not the same thing as saying that it is a bad thing to reference or to use, and this note and its extremely intrusive styling make it seem like this document should never again see the light of day.

Please change whatever it is that's doing this to the specs. It's fine to have a brightly colored note in the Status section mentioning that the document is unmaintained, if it is in fact unmaintained, but it is not appropriate to treat it the same way as out-of-date publications which have been replaced with newer ones.

(In the case of css-speech in particular, we do not think it is a bad spec. It was transitioned to a note only because there wasn't sufficient interest in speech rendering to progress to REC at the time, and if that interest increases again this spec would be a great place to start. When the CSSWG has a bad spec to retire, we delete its contents so that it is clear that it should not be re-used.)

**Comment by deniak** [comment]
@fantasai, I believe the note was added after @frivoal requested to mark [the retired obsolete notes as such](https://lists.w3.org/Archives/Public/www-style/2018Sep/0003.html).
I'll be ok to just add a sentence in the SotD instead of the red popup if everyone's ok with that.

/cc @plehegar @svgeesus @xfq

**Comment by frivoal** [comment]
For the CSS Profiles, we did talk about retiring them, and [resolved on doing it](http://www.w3.org/mid/CADhPm3tAraRSfq+aH+m0x3no+KSk-ALPyxZWnersGi8x6BztWQ@mail.gmail.com). We can discuss the exact style for doing that, but overall I don't really have an issue with that happened to them.

I did **not** request that CSS Speech got the same treatment, and I don't believe we have a resolution for doing that. I could not find any resolution for republishing as a Note, and much less one for retiring that Note. The last resolution I can find on CSS Speech is a [2017 resolution to republish as CR](https://lists.w3.org/Archives/Public/www-style/2017Feb/0069.html), and that never seems to have happened. There are various mentions by @fantasai (see [[1]](https://lists.w3.org/Archives/Member/w3c-css-wg/2017AprJun/0142.html) [[2]](http://www.w3.org/mid/90cfbad9-8c54-d8dd-a7eb-dfe04a7fc3eb@inkedblade.net) [[3]](https://www.w3.org/mid/f86cf831-f91e-2a61-0324-1b19df69f895@inkedblade.net)) during 2017 of that publication as CR being "Blocked in Publication Queue". The next things I can find are [an early 2018 commit by @svgeesus preparing for publication as a Note](https://github.com/w3c/csswg-drafts/commit/de0372460949e9511833633830c8b33ad1367e62), and [one by @xfq adding the obsoletion notice in late 2018](https://github.com/w3c/csswg-drafts/commit/549b0e470e8744bf87fabb3fea736365ad1ac1be). I also cannot find a trace in https://github.com/w3c/transitions of transitioning CSS Speech from CR to Note, or on [www-style](https://lists.w3.org/Archives/Public/www-style/), or on [w3c-css-wg](https://lists.w3.org/Archives/Member/w3c-css-wg/), or on [the CSSWG's Blog](https://www.w3.org/blog/CSS/)

Even if it might not be entirely crazy to publish as a Note due to the persisting lack of implementations, I am curious to how we went from resolving to publish as CR to actually publishing as a Note, and a retired one at that.

Also, the sentence used in the abstract of CSS Speech is inappropriate:
> This specification is obsolete. Please see the latest CSS Snapshot for the specifications that make up CSS.

That sentence was discussed for the purpose Profiles, and it is appropriate for them as they attempted to define what was or wasn't part of various profiles of CSS, and the CSS Snapshot is the newer document which has taken over the responsibility of defining what is part of CSS. But it would not be terribly useful for CSS Speech.

**Comment by frivoal** [comment]
(Also, I am far from sure this is the right repo for this issue)

**Comment by xfq** [comment]
I'm not sure about the story behind the CSS Speech CR/Note, because that had happened before I joined the CSSWG. I added the obsoletion notice just because it's [published](https://www.w3.org/TR/2018/NOTE-css3-speech-20180605/) as a Note, so I thought it was meant to be retired, like the various profiles.

I apologize for that.

**Comment by plehegar** [comment]
Looking back at the history here, Chris requested publication as a WG Note, on June 4, 2018.:
[[
* Evidence that publication is in accordance with expectations
Previously published as a Candidate Recommendation in 2012, this 
specification is being republished with a minor update and moving to WG 
Note since there are no current or planned implementations that we are 
aware of.
]]
I asked if the Note needed to be marked as Retired and Chris confirmed it.

Now, Chris and I discussed pubrules issues related to the publication of this module as a Note in February 2018. And, it seems it was blocked in "publication process" for a few months in 2017 following the CSS WG CR decision that you found.

So, as far as I can tell, we got mixed up and should not have publish the document as a Note, but I could miss something. 

@astearns , do you concur that this module needs to be republished as a CR?

**Comment by astearns** [comment]
I can't find discussion that would have changed the CR publication to a Note, but given the continued lack of implementations it seems like an appropriate state for the spec to be in (without the 'obsolete' part).

**Comment by plehegar** [comment]
So, at the minimum, we should remove the warning (I'm guessing we can do so without republishing).

**Comment by plehegar** [comment]
I removed the warning from the document.

I don't believe that retired documents are automatically treated as do-not-read. They are not listed in the default view page of /TR, but one can find them under retired. The red box is independent of the "Retired" status.

**Comment by frivoal** [comment]
> I can't find discussion that would have changed the CR publication to a Note, but given the continued lack of implementations it seems like an appropriate state for the spec to be in

As long as the spec is not being implemented, there's very little feedback coming in, and not much maintenance to do, but that neither means the WG thinks the spec is a bad idea, that we think people should not implement it, or that we wouldn't deal with issues if they were filed. To the best of my knowledge, the WG continues to believe that implementing this would be the right thing to do.

If you wanted to argue with the WG that we should abandon the spec, it is possible that we'd agree, but last time we did check, the resolution was (re)publish a CR, not a Note, and decision to publish is a group decision, not a chair decision.

Moreover, while we could revive the spec from Note to the REC track if/when implementations get going, keeping it as a Note in the meanwhile has undesirable Patent Policy implications:
* Republishing as a CR, as resolved by the WG, would trigger an exclusion opportunity, which would secure a commitment to license on the latest text by current WG members.
* Keeping it as a Note means that the provisions of the Patent Policy that secure commitments to license when Member companies join or leave the Group don't apply.

The combination of the "this is abandoned" message that is sent by the status being Note and the lack of coverage by the Patent Policy is discouraging implementation, which is the opposite of the (latest known) will of the WG.

TL;DR: Please republish as a CR :)

**Comment by fantasai** [comment]
@xfq You missed the obsoletion notice on /TR ...

**Comment by xfq** [comment]
I can create a transition request (for CR) per group decision mentioned above.

Can we show that the spec has received wide review in order to move to CR?

**Comment by frivoal** [comment]
@xfq, it has already been in CR, we had a resolution to do a CR update after having made some minor fixes in response to horizontal review.
